client/js/global.js

```diff
@@ -17,26 +17,75 @@ const $ = require('jQuery');
  * @returns {fmOverlay}
  */
 function fmOverlay() {
+  const $window = $(window);
   const $overlay = $('#fm-overlay');
   const $overlayWrapper = $overlay.children('.fm-overlay-wrapper');
+  const $overlayFade = $overlay.children('.fm-overlay-fade');
+  const $overlayImage = $overlay.find('img', '.fm-image');
```

Comment:
Would this also grab an inlined image from a rich text editor? If so, we want to make sure we don't bork any inline styles those images may have received.